.github/workflows/test_policy_sql.yml

```diff
@@ -0,0 +1,83 @@
+name: SQL Policy Validation Test
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+env: 
+  CGO_ENABLED: 0
+  CQ_NO_TELEMETRY: 1
+
+jobs:
+  SQLPolicyTest:
+    strategy:
+      matrix:
+        dbversion: [ "postgres:10" ]
+        go: [ "1.17" ]
+        platform: [ ubuntu-latest ] # can not run in macOS and widnowsOS
+    runs-on: ${{ matrix.platform }}
+    services:
+      postgres:
+        image: ${{ matrix.dbversion }}
+        env:
+          POSTGRES_PASSWORD: pass
+          POSTGRES_USER: postgres
+          POSTGRES_DB: postgres
+        ports:
+          - 5432:5432
+        # Set health checks to wait until postgres has started
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+    steps:
+      - name: Check out code into the policy directory
+        uses: actions/checkout@v3
+
+      - name: Download Cloudquery
+        run: |
+          curl -L https://github.com/cloudquery/cloudquery/releases/latest/download/cloudquery_Linux_x86_64 -o cloudquery
+          chmod a+x cloudquery
+
+      - name: Set up Go 1.x
+        uses: actions/setup-go@v3
+        with:
+          go-version: ${{ matrix.go }}
+
+      - uses: actions/cache@v3
+        with:
+          # In order:
+          # * Module download cache
+          # * Build cache (Linux)
+          # * Build cache (Mac)
+          # * Build cache (Windows)
+          path: |
+            ~/go/pkg/mod
+            ~/.cache/go-build
+            ~/Library/Caches/go-build
+            ~\AppData\Local\go-build
+          key: ${{ runner.os }}-go-${{ matrix.go }}-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+            ${{ runner.os }}-go-${{ matrix.go }}-
+
+      # All bash commands are in a single step so that we can see provider log which runs in the background.
+      - name: Run test
+        run: |
+          set -o xtrace
+
+          go run ./main.go &
+
+          # Wait until .cq_reattach exists
+          bash -c "until test -f .cq_reattach; do echo 123; sleep 1; done"
+
+          export CQ_REATTACH_PROVIDERS=.cq_reattach
+          ./cloudquery provider sync aws --config ./test/policy_cq_config.yml --enable-console-log
```

Comment:
I would do it in two steps as will be easier to understand what failed. Also I think it should be possible to build the table without the cloudquery binary.

Reply:
@yevgenypats done in [`f0ab33c` (#1220)](https://github.com/cloudquery/cq-provider-aws/pull/1220/commits/f0ab33c64a2d37d84fea5532133748a109a48ff5) (and [`7327b09` (#1220)](https://github.com/cloudquery/cq-provider-aws/pull/1220/commits/7327b094727ab0bebbb688db21a7ce29de825bf9))